lib/entities/ui-extension.ts

```diff
@@ -5,7 +5,15 @@ import enhanceWithMethods from '../enhance-with-methods'
 import { createUpdateEntity, createDeleteEntity } from '../instance-actions'
 import { EntryFields } from './entry-fields'
 import { wrapCollection } from '../common-utils'
-import { DefaultElements, MetaSysProps } from '../common-types'
+import { DefaultElements, MetaSysProps, QueryOptions } from '../common-types'
+
+export interface GetUiExtensionsQuery extends QueryOptions {
```

Comment:
I wouldn't extend `QueryOptions`. Only `stripSrcdoc` and `sys.id[in]` are supported in `/extensions`